packages/form-js-viewer/src/render/components/form-fields/Image.js

```diff
@@ -0,0 +1,62 @@
+import { isArray, isObject } from 'min-dash';
+import { useContext } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import { useImageSource } from '../../hooks/useImageSource';
+
+import {
+  formFieldClasses,
+  prefixId,
+  safeImageSource
+} from '../Util';
+
+import ImagePlaceholder from './icons/ImagePlaceholder.svg';
+
+const type = 'image';
+
+
+export default function Image(props) {
+  const {
+    field
+  } = props;
+
+  const {
+    alt,
+    id
+  } = field;
+
+  const { source } = useImageSource(field);
+  const safeSource = safeImageSource(source);
+
+  const { formId } = useContext(FormContext);
+
+  // todo(pinussilvestrus): what to display if image broken?
```

Comment:
Should we answer that question or have we already accepted something for now?  Is this PR aimed at dev or the sprint branch, because that changes the standards :D.

Reply:
It's an oversight (sorry!), let's remove the comment. We agreed to simply display the browser default on broken images.